From: Pavel Sobolev <contact@paveloom.dev>
Subject: [PATCH] Don't build examples.

This also allows us to bypass the error caused by `--disable-static`
being enabled, see the attached bug ticket.

Bug: https://github.com/google/ios-webkit-debug-proxy/issues/399

Signed-off-by: Pavel Sobolev <contact@paveloom.dev>

--- a/Makefile.am
+++ b/Makefile.am
@@ -2,4 +2,4 @@
 # Copyright 2012 Google Inc. wrightt@google.com
 
 AUTOMAKE_OPTIONS = foreign
-SUBDIRS = src include examples
+SUBDIRS = src include

--- a/configure.ac
+++ b/configure.ac
@@ -87,7 +87,7 @@ fi
 
 AC_CHECK_FUNCS([memmove memset regcomp select socket strcasecmp strncasecmp strchr strdup strndup strrchr strstr strtol strcasestr getline])
 
-AC_CONFIG_FILES([Makefile src/Makefile include/Makefile examples/Makefile])
+AC_CONFIG_FILES([Makefile src/Makefile include/Makefile])
 
 CFLAGS="${CFLAGS} -Wall -Werror"
